packages/block-library/src/site-logo/edit.js

```diff
@@ -73,7 +73,7 @@ const SiteLogo = ( {
 	const [ { naturalWidth, naturalHeight }, setNaturalSize ] = useState( {} );
 	const [ isEditingImage, setIsEditingImage ] = useState( false );
 	const { toggleSelection } = useDispatch( blockEditorStore );
-	const classes = clsx( 'custom-logo-link' );
+	const classes = 'custom-logo-link';
```

Comment:
I think we can go a step further and just inline such cases without the need to maintain a constant.